package.json

```diff
@@ -38,7 +38,8 @@
     "@aws-cdk/aws-apigatewayv2-authorizers-alpha": "^2.104.0-alpha.0",
     "@aws-cdk/aws-apigatewayv2-integrations-alpha": "^2.104.0-alpha.0",
     "@aws-cdk/aws-cognito-identitypool-alpha": "^2.104.0-alpha.0",
-    "aws-cdk-lib": "2.104.0",
+    "@aws-solutions-constructs/aws-eventbridge-lambda": "^2.46.0",
+    "aws-cdk-lib": "2.105.0",
```

Comment:
We want to be consistent on versioning and we usually bump on dedicated PR after testing deployments of the all infra with all components before doing so

Any specific reason to bump to 2.105 here? can this be left at 2.104 for this PR?

Reply:
@bigadsoleiman I agree with you about version changes. 
I introduced the AWS solutions construct which had peer dependencies on v2.105. 
Alternatively, I can remove the construct and simply create an event bridge rule manually. There's little effort to make that change and I can revert the version bump.

Reply:
If you need that specific version because the feature you are using is only available there, ensure you bump the versions of all the CDK packages.